### PR TITLE
Add invoice item put and delete paths

### PR DESCRIPTION
**Pull request by subenson**

### CODE REVIEW

File: openapi/paths/invoices@{id}@items.yaml [hunk 1]
```diff
@@ -82,3 +82,72 @@ post:
...
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/Invoices/InvoiceItem.yaml
+    description: InvoiceItem resource.
+    required: true
+  responses:
+    '201':
+      description: InvoiceItem was created.
```
Comment (ashkarpetin) [inline openapi/paths/invoices@{id}@items.yaml:102]: Is it correct?

Comment (ashkarpetin) [reply openapi/paths/invoices@{id}@items.yaml:102]: 201 for create, 200 for update

**Comment by ZengJunyong** [comment]
When updating/deleting an invoice item, the invoice item id(not invoice id) is required, I don't see it [here](https://preview.redoc.ly/rebilly-core/update-invoice-items/tag/Invoices#operation/DeleteInvoiceItem)

File: openapi/paths/invoices@{id}@items@{itemId}@.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/Invoices/InvoiceItem.yaml
+    description: InvoiceItem resource.
+    required: true
+  responses:
+    '200':
+      description: InvoiceItem was created.
```
Comment (ZengJunyong) [inline openapi/paths/invoices@{id}@items@{itemId}@.yaml:27]:
```suggestion
      description: InvoiceItem was updated.
```

**Review by ZengJunyong — DISMISSED** [review]

**Review by ZengJunyong — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**